### PR TITLE
fun: fix int type edge values consts

### PR DESCRIPTION
**Pull request by ArtemkaKun**
This PR needed to make tests pass in https://github.com/vlang/v/pull/19809

## Summary by CodeRabbit

- **Bug Fixes**
	- Corrected a variable reference error in the binomial function, enhancing its accuracy and reliability.

### CODE REVIEW

File: fun/fun.v [hunk 1]
```diff
@@ -65,7 +65,7 @@ pub fn uint_binomial(n_ u64, k_ u64) u64 {
 	mut c := u64(1)
 	for i := u64(1); i <= k; i++ {
 		c_i := c / i
-		max_n := math.max_u64 / n
+		max_n := max_u64 / n
```
Comment (ulises-jeremias) [inline fun/fun.v:68]: this change is wrong and it is breaking the CI. We need to get the value from `math` 👀 why are you proposing this change? are you getting any error?

**Review by ulises-jeremias — CHANGES_REQUESTED** [review]

**Review by ulises-jeremias — APPROVED** [review]
